src/Components/Common/Tables.js

```diff
@@ -84,10 +84,10 @@ export const mapClustersToRows = (clusters) =>
         </Link>
       </span>,
       cluster.total_hit_count,
+      cluster.hits_by_total_risk?.[4] || 0,
       cluster.hits_by_total_risk?.[3] || 0,
       cluster.hits_by_total_risk?.[2] || 0,
       cluster.hits_by_total_risk?.[1] || 0,
```

Comment:
This is getting key '1' as string or the first value in the object?

Reply:
Yes
It goes cluster -> hits_by_total_risk -> ?. check if the index 1/2/3/4 exist if it exist it will pull up the data

Reply:
But we should not check index but keys. The response `{ "1": 3, "2": 9, "3": 3, "4": 0 }` is a mapping is does not guarantee order